services/merak-network/activities/vnet_create_activity.go

```diff
@@ -160,10 +151,15 @@ func doSg(sg *pb.InternalSecurityGroupInfo, sgID string, projectId string) (stri
 
 func VnetCreate(ctx context.Context, netConfigId string, network *pb.InternalNetworkInfo, wg *sync.WaitGroup, returnMessage chan *pb.ReturnNetworkMessage, projectId string) (*pb.ReturnNetworkMessage, error) {
 	log.Println("VnetCreate")
```

Comment:
@yanmo96 Did you use `ctx`, `network`, and `returnMessage` passing in parameters in this function?
If not, please remove them.

Reply:
removed now